src/main/resources/config_file_documentation.txt

```diff
@@ -58,12 +58,12 @@ Config file must be valid JSON format. Examples can be found under the test fold
         example: /Users/test_data/2015-portland-mayor-cvr.xlsx
         value: string of length [1..1000]
 
-      "firstVoteColumnIndex" required
+      "firstVoteColumnIndex" required for Excel files
```

Comment:
"for Excel files" seems a little vague. Maybe "required unless cast-vote records are in JSON format"? This applies to all instances where "Excel" is used here.